### PR TITLE
[RELEASE-1.4] [SRVKE-1350] Use env configurator

### PR DESCRIPTION
**Pull request by skonto**
- This seem to do the trick by falling back to env configurator. If config-observability is updated a new instance is launched with the new env config.

### CODE REVIEW

**Comment by skonto** [comment]
Will cherry-pick on 1.5+

File: cmd/mtping/main.go [hunk 1]
```diff
@@ -39,17 +39,5 @@ func main() {
 
 	ctx = adapter.WithController(ctx, mtping.NewController)
 	ctx = adapter.WithHAEnabled(ctx)
-
```
Comment (matzew) [inline cmd/mtping/main.go:42]: have we discussed this potential issue, upstream?

Comment (matzew) [reply cmd/mtping/main.go:42]: we could potentially look for a thread upstream.

(I am not blocking the fix here for our product ... FWIW)

**Comment by matzew** [comment]
/retest
....

**Comment by mgencur** [comment]
The CI was disabled for this repo. Please send this PR against github.com/openshift-knative/eventing. Thanks and sorry for inconvinience.

**Comment by skonto** [comment]
I guess we only need it for 1.5 so I can close it.

**Pull request closed without merging**